main.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'app'
+
+def display_list
+  action_list = {
+    '1': 'List all books',
+    '2': 'List all people',
+    '3': 'Create a person',
```

Comment:
* Kindly fix this error i get when i try to create a person 
![image](https://user-images.githubusercontent.com/52497246/224379404-b36c24f9-2619-4b55-ade8-1aeb19dd0b66.png)
![image](https://user-images.githubusercontent.com/52497246/224379432-5312eb83-a53c-4b88-93a4-8e5c3b5c7f9a.png)